### DISCUSSION TITLE
Display lower res image while loading full size!

### DISCUSSION THREAD

**Discussion by IdreesM0** [Docile-Alligator/Infinity-For-Reddit#939] [Ideas]
Scrolling through the feed, for **blurred Spoiler and NSFW images**, most of the times I would want to quickly glance through the image (which would be the low res preview version) and if I like it, only then I can go through the waiting for it to load full resolution in order to throughly enjoy it.

As of now, for any blurred images, the only way to see them is to tap and then wait for the full res image to load (which for some large images takes quite a while). 

The best approach, in my humble opinion, for all images, will be to show the low res preview when we tap the image while it loads the full res in the meanwhile. The progress for full image loading could be shown somewhere on the screen.

Would absolutely love this update in the app. 
Many Thanks Mr Developer (:

**Comment by Docile-Alligator** [comment]
Thank you! Great idea!
